src/main/java/frc/robot/Robot.java

```diff
@@ -32,18 +32,39 @@ public void robotInit() {
         shooterSystem = new ShooterSystem();
         intakeSystem = new IntakeSystem();
         armSystem = new ArmSystem();
-        xboxController = new XboxController(0);
+        xboxController = new XboxController(RobotMap.XBOX_CONTROLLER_PORT);
 
-        POVButton dPadUp = new POVButton(xboxController, 0);
-        POVButton dPadDown = new POVButton(xboxController, 180);
+        armSystem.setDefaultCommand(
+                new ParallelCommandGroup(
+                        new ParallelDeadlineGroup(
+                                new WaitUntilCommand(() -> armSystem.reachedATargetAngle(RobotMap.ARM_SHOOTER_ANGLE)),
+                                new InstantCommand(() -> intakeSystem.slowIn())
```

Comment:
this is useful for any arm-to-shooter motion, including other command groups, we will need to share this between them. But I guess it can wait for later